src/components/header/HeaderFromLecture.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { links, user } from '../../constants';
+import Logo from '../../assets/react.png';
+
+import { UserInfo } from '../user-info/UserInfoFromLecture';
+
+import './Header.scss';
+
+export const Header = (props) => {
+  console.log(links);
+  return (
+    <div className="may-header navbar">
+      <img src={Logo} className="may-header-logo" />
+
+      <div className="may-header-links-wrapper">
+        <div className="nav-item">
```

Comment:
good job
if you want, you can read how to use array function map in order to simplify logic below. However, you haven't covered that right now and this is up to you.

you can google something like: react how to render list with map function